src/test/java/teammates/test/cases/ui/InstructorFeedbackQuestionEditActionTest.java

```diff
@@ -1202,6 +1202,73 @@ public void testExecuteAndPostProcessRubricQuestion() throws Exception {
         FeedbackSessionsLogic.inst().deleteFeedbackSessionCascade(fs.feedbackSessionName, fs.courseId);
     }
 
+    @Test
+    public void testExecuteAndPostProcessResponseRate() throws Exception {
+        
+        removeAndRestoreTypicalDataInDatastore();
+        
+        gaeSimulation.loginAsInstructor(dataBundle.instructors.get("instructor1OfCourse1").googleId);
+
+        FeedbackSessionsLogic fsLogic = FeedbackSessionsLogic.inst();
+        FeedbackQuestionsLogic fqLogic = FeedbackQuestionsLogic.inst();
+
+        FeedbackSessionAttributes fs = dataBundle.feedbackSessions.get("session1InCourse1");
+
+        // Check response rate before editing question 1
+        fs = fsLogic.getFeedbackSession(fs.feedbackSessionName, fs.courseId);
+        assertEquals(4, fsLogic.getFeedbackSessionDetails(fs).stats.submittedTotal);
```

Comment:
I think we can check for both <code>submittedTotal</code> and <code>expectedTotal</code>